quality.gradle

```diff
@@ -78,11 +78,13 @@ task pmd(type: Pmd, group: 'verification') {
     }
 }
 
-android {
-    lintOptions {
-        abortOnError true
-        xmlReport true
-        htmlReport true
-        lintConfig file("${project.rootDir}/AndroidCodeQualityConfig/quality/lint/lint.xml")
+afterEvaluate {
```

Comment:
That's two different problem.

1. You don't want add apply for every module, so you add below code into root project build.gradle:

``` gradle
subprojects {
    apply from: "$rootProject.projectDir/AndroidCodeQualityConfig/quality.gradle"

    afterEvaluate {
        check.dependsOn 'checkstyle', 'findbugs', 'pmd', 'lint'
   }
}
```

and remove `check.dependsOn 'checkstyle', 'findbugs', 'pmd', 'lint'` from quality.gradle.

2. Why do you open that pr?